src/omero_zarr/raw_pixels.py

```diff
@@ -78,18 +76,62 @@ def planeGen() -> np.ndarray:
         longest = longest // 2
         level_count += 1
 
+    return add_raw_image(
+        planes=planes,
+        size_x=size_x,
+        size_y=size_y,
+        size_z=size_z,
+        size_c=size_c,
+        size_t=size_t,
+        d_type=d_type,
+        parent=parent,
+        level_count=level_count,
+        cache_dir=cache_dir,
+        cache_file_name_func=get_cache_filename,
+    )
+
+
+def add_raw_image(
+    *,
+    planes: Iterator[np.ndarray],
```

Comment:
Ideally this would be a callable of the form `f(z: int, c: int, t: int) -> np.array`, but this has to fit in with OMERO's BlitzGateway plane generator function so instead it has to return the planes in the hard-coded zct_list order.